src/parser.rs

```diff
@@ -405,18 +405,27 @@ impl<'tokens, 'src> Parser<'tokens, 'src> {
     let expression = if self.accepted_keyword(Keyword::If)? {
       self.parse_conditional()?
     } else {
-      let value = self.parse_value()?;
-
+      let value = self.parse_value();
       if self.accepted(Slash)? {
+        // Enables Slash operator to build absolute paths, like `/ "Users"`, by
+        // setting a default empty string on the lhs if the resulting expression
+        // is not Ok.
+        let value = value.unwrap_or(Expression::StringLiteral {
```

Comment:
I think this `unwrap_or` will swallow errors. What if `self.parse_value()` returns an error after skipping over some tokens?

Maybe something like this instead?

```rust
    let expression = if self.accepted_keyword(Keyword::If)? {
      self.parse_conditional()?
    } else if self.accepted(Slash)? {
        let lhs = None;
        let rhs = Box::new(self.parse_expression()?);
        Expression::Join { lhs, rhs }
    } else {
      let value = self.parse_value()?;
      
      if self.accepted(Slash)? {
        let lhs = Some(Box::new(value));
        let rhs = Box::new(self.parse_expression()?);
        Expression::Join { lhs, rhs }
      } else if self.accepted(Plus)? {
        let lhs = Box::new(value);
        let rhs = Box::new(self.parse_expression()?);
        Expression::Concatenation { lhs, rhs }
      } else {
        value
      }
    };
```

Reply:
It doesn't actually accept an option, but you can change it in this PR so it does!